internal/local/dependencies_test.go

```diff
@@ -87,3 +87,49 @@ func TestDependenciesFind(t *testing.T) {
 		})
 	}
 }
+
+func TestDependenciesFindPackageJSON(t *testing.T) {
+	wd, wdErr := os.Getwd()
+	assert.Nil(t, wdErr)
+
+	testRoot := filepath.Join(wd, "testdata/dependencies")
+
+	t.Run("should return an empty data when run outside a project directory", func(t *testing.T) {
+		deps, err := FindPackageJSON(testRoot)
+		assert.Nil(t, err)
+		assert.Equal(t, Dependencies{}, deps)
+	})
+
+	t.Run("should return an error when a project has no package.json file", func(t *testing.T) {
+		dir := filepath.Join(testRoot, "empty")
+
+		_, err := FindPackageJSON(dir)
+		assert.NotNil(t, err)
+		assert.Equal(t,
+			err.Error(),
+			fmt.Sprintf("package.json not found at '%s/functions'", dir),
+		)
+	})
+
+	t.Run("Should find a package.json", func(t *testing.T) {
```

Comment:
[nit] errant capitalization here
```suggestion
	t.Run("should find a package.json", func(t *testing.T) {
```